packages/fxa-auth-server/lib/luaScripts/pruneSessionTokens_1.lua

```diff
@@ -0,0 +1,29 @@
+-- https://github.com/mozilla/fxa-auth-server/pull/2254 for packing scheme info
```

Comment:
Packing scheme doesn't seem relevant to this script.

Reply:
true